crates/bitwarden-ssh/Cargo.toml

```diff
@@ -22,14 +22,19 @@ wasm = [
 
 [dependencies]
 bitwarden-error = { workspace = true }
+ed25519 = { version = "2.2.3", features = ["pkcs8"] }
+pkcs8 = { version = "0.10.2", features = ["encryption"] }
 rand = "0.8.5"
+rsa = "0.9.7"
 serde.workspace = true
 ssh-key = { version = "0.6.7", features = [
     "ed25519",
     "encryption",
     "rsa",
     "getrandom",
-] }
+    "rand_core",
+    "std",
```

Comment:
Some of these flags seems unecessary. `std` brings in several eliptic curve dependencies.

Reply:
Removed the dep and the tests still run. Also moved the test keys to `resources` and excluded it.